common/Timer.h

```diff
@@ -58,6 +58,9 @@ class Timer {
 
     static microseconds clock_gettime_coarse();
 
+    // Disable dropping timers with duration below clock_threshold_coarse for tests
+    void disableDroppingShortTimersForTests();
```

Comment:
Let's land usage in the same PR?
Will make it easier to discuss motivation and thus if this is the right approach.

Reply:
(Changing such global configs in tests feels problematic for me, but I lack context to know what are we looking to address)

Reply:
Context: We have tests that ensure that LSP is reporting timing metrics correctly.

However, timing metrics are dropped if they are below a certain threshold.

I've inserted sleeps into some tests to ensure that expected timers are reported, but it's very brittle.

This mechanism is intended to let us _disable_ the timer optimization so we can remove those sleeps and make these few metrics tests robust and fast.